internal/store/postgres/asset_repository.go

```diff
@@ -402,20 +401,21 @@ func (r *AssetRepository) UpsertPatch( //nolint:gocognit
 			return nil
 		}
```

Comment:
sorry missed this on previous PR, but we can simplify this code

Reply:
updated